plots/measure.js

```diff
@@ -29,9 +30,47 @@ const ChromeLauncher = require('../lighthouse-cli/chrome-launcher.js').ChromeLau
 const Printer = require('../lighthouse-cli/printer');
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
 
-const NUMBER_OF_RUNS = 20;
+const DISABLE_DEVICE_EMULATION = false;
+const DISABLE_CPU_THROTTLING = true;
+const DISABLE_NETWORK_THROTTLING = args['disable-network-throttling'] || false;
+const KEEP_FIRST_RUN = args['keep-first-run'];
 
-const URLS = [
+// Running it n + 1 times if the first run is deliberately ignored
+// because it has different perf characteristics from subsequent runs
+// (e.g. DNS cache which can't be easily reset between runs)
+const NUMBER_OF_RUNS = (args.n || 20) + (KEEP_FIRST_RUN ? 0 : 1);
+
+const FLAGS = {
+  output: 'json',
+  disableCpuThrottling: DISABLE_CPU_THROTTLING,
+  disableNetworkThrottling: DISABLE_NETWORK_THROTTLING,
+  disableDeviceEmulation: DISABLE_DEVICE_EMULATION,
+};
+console.log('Running lighthouse with flag\n disableNetworkThrottling: ', FLAGS.disableNetworkThrottling);
+
+const SUBSET = [
```

Comment:
running plots/ this weekend, it was very handy to split this to loading from file so I could quickly switch between a couple of different sets of test sites. If you're adding `yargs` anyways, it would be great to support loading a list of URLs from a path passed in as a CLI option :)